app.py

```diff
@@ -207,6 +209,28 @@ def save_identity():
     return jsonify({})
 
 
+# End Point example which verifies payload signature
+@app.route('/v1/signed_payload', methods=['POST'])
+@validate_json
+def signed_payload():
+    payload = request.get_json(force=True)
+    signature_base64_encoded = request.headers.get('signature')
+    identity = payload['identity']
```

Comment:
- Some payloads does not have `identity` attribute
- We could add 2 headers `identity` & `signature`

Reply:
I would suggest to use single predefined header Authorization: with signature bearer, to avoid introducing unconventional custom headers [https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization](Example)

Reply:
- Some payloads does not have `identity` attribute
- It's each endpoint specific, so each endpoint will do:
```
caller_identity = get_caller_identity()
if payload['provider_id'] != caller_identity {
    showError("Cant publish proposal for other identity")
}
```

- We could add 2 headers identity & signature

Reply:
moved `identity` to request header

Reply:
Added HTTP Authorization request header:
Authorization: Signature <base64_encoded_signature>